distrib/initscripts/rc.openrc.tmpl

```diff
@@ -84,25 +84,25 @@ stop () {
 		eend $?
 	fi
 
-	if [ "${TIMELORD_RUN}" = "yes" ]; then
-		ebegin "Stopping timelord"
-		start-stop-daemon --stop --quiet --exec :SBINDIR:/timelord
-		eend $?
-	fi
-
-	if [ "${A2BOOT_RUN}" = "yes" ]; then
-		ebegin "Stopping a2boot"
-		start-stop-daemon --stop --quiet --exec :SBINDIR:/a2boot
-		eend $?
-	fi
-
 	if [ "${ATALKD_RUN}" != "no" ]; then
 		if [ "${PAPD_RUN}" = "yes" ]; then
 			ebegin "Stopping papd"
 			start-stop-daemon --stop --quiet --exec :SBINDIR:/papd
 			eend $?
 		fi
 
+		if [ "${TIMELORD_RUN}" = "yes" ]; then
+			ebegin "Stopping timelord"
+			start-stop-daemon --stop --quiet --exec :SBINDIR:/timelord
+			eend $?
+		fi
+
+		if [ "${A2BOOT_RUN}" = "yes" ]; then
+			ebegin "Stopping a2boot"
```

Comment:
Minor thing, but it would look nicer to indent the "Stopping" messages like you did with the "Starting" messages.